### DISCUSSION TITLE
mspider/user-guide/service-mesh/index

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2783] [Announcements]
# mspider/user-guide/service-mesh/index

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/service-mesh/index.html

**Comment by samzong** [comment]
托管/专有网格创建步骤页面内配图需要更新

#2782
